Solutions/CiscoUmbrella/Hunting Queries/CiscoUmbrellaHighValuesOfUploadedData.yaml

```diff
@@ -13,6 +13,6 @@ query: |
   | where EventType == "proxylogs"
   | where TimeGenerated > ago(timeframe)
   | summarize sum(DstBytes) by SrcIpAddr,DstIpAddr
-  | sort by sum_DstBytes desc
+  | sort by sum_SrcBytes desc
   | project SrcIpAddr, DstIpAddr
-  | extend IpCustomEntity = SrcIpAddr
\ No newline at end of file
+  | extend IpCustomEntity = SrcIpAddr
```

Comment:
If we are updating this query can we please add the updated entity mapping format to this template

Reply:
Helped out Rod here and added the format to support future entity mapping capabilities for hunting queries